packages/focusjail/src/useFocusJail.ts

```diff
@@ -10,11 +10,28 @@ import { composeEventHandlers, KEY_CODES } from '@zendeskgarden/container-utilit
 import tabbable from 'tabbable';
 import activeElement from 'dom-helpers/activeElement';
 
-export function useFocusJail({ focusOnMount = true, environment, focusElem, containerRef } = {}) {
+export interface IUseFocusJailProps {
+  focusOnMount?: boolean;
+  environment?: Document;
+  focusElem?: (element: HTMLElement) => any;
+  containerRef?: any;
+}
+
+export interface IUseFocusJailReturnValue {
+  getContainerProps: <T>(options?: T & React.HTMLProps<any>) => any;
```

Comment:
This should match the same structure as the other hooks. Since there are not required options I think we could do:

```tsx
getContainerProps: <T>(options?: T) => T & React.HTMLProps<any>;
```

Reply:
Thanks, updated!